libcloudforensics/providers/aws/forensics.py

```diff
@@ -206,9 +206,24 @@ def StartAnalysisVm(
   Returns:
     Tuple[AWSInstance, bool]: a tuple with a virtual machine object
         and a boolean indicating if the virtual machine was created or not.
+
+  Raises:
+    RuntimeError: If the requested provider or function is not
```

Comment:
The RuntimeError justification probably needs to be updated with the actual reason it's raised ;)